dev-support/ci/selective_ci_checks.sh

```diff
@@ -399,19 +406,18 @@ function check_needs_unit_test() {
     local pattern_array=(
         "^hadoop-ozone/dev-support/checks/_mvn_unit_report.sh"
         "^hadoop-ozone/dev-support/checks/unit.sh"
-        "pom.xml"
```

Comment:
Does this mean changing the `pom.xml` won't trigger the unit test?

Reply:
With this patch unit test will be triggered via this branch for `pom.xml` changes:

https://github.com/apache/ozone/blob/e3378947c491dbf66a30c010ac339ddd2832daf9/dev-support/ci/selective_ci_checks.sh#L466-L473

This is covered by the "pom change" test case:

https://github.com/apache/ozone/blob/e3378947c491dbf66a30c010ac339ddd2832daf9/dev-support/ci/selective_ci_checks.bats#L201-L210

BTW same applies to main (non-test) Java changes.  Now I've added a test case for that, too:

https://github.com/apache/ozone/blob/5843ab95c4f4a5c3eba3111b86da258f0f0ad499/dev-support/ci/selective_ci_checks.bats#L168-L177

Reply:
Got it! Thanks